C++ API parity: Threshold

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27309 C++ API parity: MultiheadAttention
* #27382 C++ API parity: Linear
* **#27538 C++ API parity: Threshold**

Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

pbelevich committed:
C++ API parity: Threshold

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"




[ghstack-poisoned]

yf225 reviewed (approved):
Thanks so much for the impeccable work @pbelevich!

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Threshold"


Differential Revision: [D17835415](https://our.internmc.facebook.com/intern/diff/D17835415)

[ghstack-poisoned]
